### DISCUSSION TITLE
yarn dev compile error - can't resolve 'photoswipe/dist/photoswipe.css'

### DISCUSSION THREAD

**Discussion by litecactus** [ente-io/ente#4218] [Q&A]
I'm testing ente self hosting setup via docker compose on Ubuntu 24.02 , am following steps on https://help.ente.io/self-hosting/

I run `NEXT_PUBLIC_ENTE_ENDPOINT=http://localhost:8080 yarn dev` and it fails with the following error...
```
Module not found: Can't resolve 'photoswipe/dist/photoswipe.css'
  44 | import type { AppProps } from "next/app";
  45 | import { useRouter } from "next/router";
> 46 | import "photoswipe/dist/photoswipe.css";
     | ^
  47 | import { useCallback, useEffect, useState } from "react";
  48 | import LoadingBar from "react-top-loading-bar";
  49 | import { resumeExportsIfNeeded } from "services/export";
```

![CleanShot - jOI0eL8G](https://github.com/user-attachments/assets/d70456ca-68ee-4cc8-ba13-da5b709a5472)

Any support would be appreciated.

**Comment by mnvr** [comment]
```
git submodule update --init --recursive 
```

(context is the code block for the "run (e.g) the web client" [section](https://help.ente.io/self-hosting/))
